src/generator/endpoints/elevenLabsApi.py

```diff
@@ -0,0 +1,102 @@
+import json
+import logging
+from datetime import datetime
+
+from fastapi import APIRouter, FastAPI, HTTPException, UploadFile, File
+from pydantic import BaseModel
+import elevenLabsLogic
+import os
+
+API_KEY = "a8412809ee48754b006d19396dd7eb40"
+
+# API_KEY = os.getenv("API_KEY")
+# if API_KEY == "":
+#     raise Exception("ElevenLabs API_KEY not set")
+
+testAPI = elevenLabsLogic.ElevenLabsAPI(apiKey=API_KEY)
+
+app = FastAPI()
+
+logger = logging.getLogger("generator")
+
+
+class CreateVoicePayload(BaseModel):
+    name: str
+    audio_file: UploadFile
+    description: str
+
+class SetVoicePayload(BaseModel):
+    selected_voice: str
+
+class ReadTextsPayload(BaseModel):
+    path_to_texts_file: str
+    path_to_save_files_dir: str
+    base_filename: str
+
+
+
+@app.post("/create_voice/")
```

Comment:
```suggestion
@app.post("/create_voice")
```
Dotyczy to wszystkich endpointów. Nie chcemy mieć / na koniec